### DISCUSSION TITLE
Translations in the same document.

### DISCUSSION THREAD

**Discussion by KULTI1995** [firecmsco/firecms#424] [Q&A]
Is it possible to enter translations, in the same document, instead of creating a new collection in the document?

**Comment by fgatti675** [comment]
Hi, I am moving some of your remarks to disucssions, since they are not really issues.
You can do that by using a `map` property instead of a subcollection. You are free to use any data structure you need

**Reply by KULTI1995** [reply]
Is there an example of such a solution anywhere?

**Reply by KULTI1995** [reply]
AA, can such a solution be placed in a separate panel, as it is when we introduce sub-collections to the admin panel?
